src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderList.jsx

```diff
@@ -194,21 +194,27 @@ export default function ProviderList({
                   }}
                 >
                   <span className="sr-only">show</span>
-                  <span className="va-button-link">
-                    +{' '}
+                  <span>
+                    Show{' '}
                     {Math.min(
                       communityCareProviderList.length - providersListLength,
                       initialProviderDisplayCount,
                     )}{' '}
-                    more providers
+                    more provider
+                    {Math.min(
+                      communityCareProviderList.length - providersListLength,
+                      initialProviderDisplayCount,
+                    ) > 1
+                      ? 's'
+                      : ''}
                   </span>
                 </button>
               </>
             )}
             {communityCareProviderList?.length > 0 && (
               <button
                 type="button"
-                className="vaos-appts__cancel-btn va-button-link vads-u-margin--0 vads-u-flex--0"
+                className="vaos-appts__cancel-btn usa-button-secondary vads-u-margin-right--0 vads-u-flex--0"
```

Comment:
I would remove 'vads-u-margin-right--0' from the cancel button. It displays correctly without it. I would also remove 'vads-u-flex--0'. This must be old since I couldn't find this style definition anywhere